packages/aws-cdk/lib/init-templates/app/fsharp/cdk.json

```diff
@@ -0,0 +1,4 @@
+{
+    "app": "dotnet src/HelloCdk/bin/Debug/netcoreapp2.1/HelloCdk.dll"
```

Comment:
I'm going to change the C# cdk.json template to:

"app": "dotnet run --project HelloCdk/HelloCdk.csproj"

To eliminate the need to remember to recompile the application every time before you run "cdk *". Feel free to make this change now, or I can make it when I update the C# one.

Reply:
I'll update it after the C# one. 
Do you have an opinion on my comments at the beginning regarding the solution not referencing the project and the project not referencing the project files? They're not very useful in their current state, happy to update those as well.

Reply:
Thanks for calling this out.

The C# template adds the project after generation using the add-project.hook.ts. I see you copied this file, but did not update it for the F# template. Try changing the "dotnet add" command to point to the fsproj, and see if it works. I added the comments to the lines that need to change.

Reply:
This still doesn't address the missing project files in the project file (HelloCdk.[c|f]sproj), 
could add the logic to the `add-project.hook.ts` file in another PR.

Also, `dotnet restore` is not automatically done in all IDEs so it could be added as another post template init logic like we do with npm and maven.

Reply:
I'm confused about what is missing. The add-project hook calls ```dotnet add```. If there are additional resources missing for F# on the project level, then you can either add those references into the template, or you can take care of it in the/a hook.

The ```dotnet restore``` is definitely something that could be added.

Reply:
@costleya My bad, forgot they changed it in the last version of the framework, you don't need to add files to project now. It's directory based now.

Happy to add the `dotnet restore` bit in a separate PR to not pollute this one.

Reply:
Sounds like a plan.